__tests__/app.test.js

```diff
@@ -176,6 +176,7 @@ describe("GET: /api/articles/:article_id", () => {
             votes: 100,
             article_img_url:
               "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: expect.any(Number),
```

Comment:
Well done on remembering to add this to the test, don't forget to keep your endpoints.json updated too!